### DISCUSSION TITLE
Support multiple languages

### DISCUSSION THREAD

**Discussion by crstian19** [kwchang0831/svelte-QWER#49] [Q&A]
I have a blog and I just discovered qwer and I would love to migrate my posts to qwer.

My blog is written in Spanish and English with the path /post/ and in English /en/post, is there currently no way to include different languages?

**Comment by kwchang0831** [comment]
You have total control of your site path. The path is directory based.

For example, you have the following
```
user/blogs/post/1
user/blogs/en/post/2
```
This generates 

```
http://localhost:5174/post/1
http://localhost:5174/en/post/2
```

Also, uses `language` frontmatter in your markdown. For more details see https://docs-svelte-qwer.vercel.app/writing/frontmatter#language

```
---
language: español
---
```

or 

```
---
language: English
---
```

When the blog is generated, the tags section will have the language for you to filter.
![image](https://user-images.githubusercontent.com/88667293/224523572-30e08fd5-7883-4f75-aad0-dcdfa280edcb.png)
